dataactbroker/handlers/fileHandler.py

```diff
@@ -1046,12 +1046,20 @@ def serialize_submission(submission):
     else:
         submission_user_name = sess.query(User).filter_by(user_id=submission.user_id).one().name
 
+
+    cgac = sess.query(CGAC).\
+        filter_by(cgac_code=submission.cgac_code).one_or_none()
+    if cgac:
+        agency_name = cgac.agency_name
+    else:
+        agency_name = ''
+
     return {
         "submission_id": submission.submission_id,
         "last_modified": submission.updated_at.strftime('%Y-%m-%d'),
         "size": total_size,
         "status": status,
-        "errors": submission.number_of_errors,
+        "agency": agency_name,
```

Comment:
We can shorten the above if/else block here to be "agency": cgac.agency_name if cgac.agency_name else ''. Also, I'm not sure we want to display just a blank value if there's no agency associated with a submission (although that shouldn't be the case for a submission, since it should always have a submission), but we may want to return None instead of '' which will translate to null in the response JSON.

Either way, that value needs to probably be translated to something more user friendly. I'd refer to Kaitlin on what they'd like to see in that scenario.